projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
+
+Once you've obtained your personal test credentials you can start with the development of your integration with the UiTdatabank Entry API on our test environment.
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` and provide us the identifiers of the created test events. We will then validate your integration as soon as possible. 
+
+In this validation process we assess the integration in two manners:
+1. Is the **created data** qualitative, relevant and valuable for public agendas such as www.uitinvlaanderen.be?
+2. Is the **integration made robust**, e.g. is it possible to update and delete content through the integration?
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
+
+### Checklist for a succesfull integration
+#### Mandatory fields
+- [ ] **name**: only use plain text 
```

Comment:
The linter is not a fan of these `- [ ]` checkboxes (this causes the "Found reference to undefined definition" in the logs, because it interprets them as links to local anchors I think). I don't think these are standard Markdown but an extension added on top of Markdown. I'll check if I can add a plugin so the linter parses them as valid.

Reply:
Additionally, maybe making them "checked" is clearer? (So `- [x]` instead of `- [ ]`).
Because now they're just blank squares in the docs:

![Screen Shot 2022-10-25 at 17 42 51](https://user-images.githubusercontent.com/959026/197819813-d60d9268-a61a-467b-bce9-1a32d5826a50.png)

And checked the look like:

![Screen Shot 2022-10-25 at 17 43 11](https://user-images.githubusercontent.com/959026/197819887-753d7a9e-f459-4195-899f-ce2009f8816e.png)

(Note that in both cases they're disabled so they cannot be checked/unchecked by the reader)

Reply:
I tried to find a quick solution, but the plugin that exists for this (`remark-gfm`) is not easy to get working and it also reports additional warnings/errors. So it's best to do that in a later, separate PR. Can we convert this to just a bullet list for now?

Reply:
`- [ ]` is part of the "stoplight flavored markdown". I'll correct it to an unnumbered list

Reply:
Yep Stoplight Flavored Markdown is an extension of GitHub Flavored Markdown, which added this `- [ ]` syntax. However the linter does not understand it right now, so I'll need to add support for it in another PR first but it will take some time.